playkit/src/main/java/com/kaltura/playkit/drm/DeferredDrmSessionManager.java

```diff
@@ -123,6 +142,32 @@ public DrmSession acquireSession(Looper playbackLooper, @Nullable EventDispatche
         return drmSessionManager.acquireSession(playbackLooper, eventDispatcher, format);
     }
 
+    private DrmSessionManager getDRMSessionManager(DrmCallback drmCallback) {
+        DrmSessionManager drmSessionManager;
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2 && isForceWidevineL3Playback) {
+            drmSessionManager = new DefaultDrmSessionManager.Builder()
+                    .setMultiSession(true) // key rotation
+                    .setPlayClearSamplesWithoutKeys(allowClearLead)
+                    .setUuidAndExoMediaDrmProvider(
+                            MediaSupport.WIDEVINE_UUID,
+                            uuid -> {
+                                try {
+                                    FrameworkMediaDrm frameworkMediaDrm = FrameworkMediaDrm.newInstance(MediaSupport.WIDEVINE_UUID);
+                                    frameworkMediaDrm.setPropertyString(SECURITY_LEVEL_PROPERTY, WIDEVINE_SECURITY_LEVEL_3);
```

Comment:
you have to build it also for the previous provider I think in case of default
lets discuss it

Reply:
Done